service/controller/handler/configversion/create.go

```diff
@@ -0,0 +1,85 @@
+package configversion
+
+import (
+	"context"
+
+	"github.com/ghodss/yaml"
+	"github.com/giantswarm/apiextensions/v3/pkg/annotation"
+	"github.com/giantswarm/microerror"
+
+	controllerkey "github.com/giantswarm/config-controller/service/controller/key"
+)
+
+func (h *Handler) EnsureCreated(ctx context.Context, obj interface{}) error {
+	app, err := controllerkey.ToAppCR(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	value, ok := app.GetAnnotations()[annotation.ConfigVersion]
+	if ok {
+		h.logger.Debugf(ctx, "App CR %#q is already annotated with %#q: %s", app.Name, annotation.ConfigVersion, value)
```

Comment:
There is no need to mention app.Name in the logs. There is already `object` key injected into all handler log messages.